java/src/main/java/com/logicalclocks/hsfs/Feature.java

```diff
@@ -196,6 +196,10 @@ public Filter in(Collection<?> collection) {
     return new Filter(this, SqlFilterCondition.IN, jsonArray.toString());
   }
 
+  public Filter lk(Feature value) {
```

Comment:
Let's call it also `like` like in python?

Reply:
Ok will change. I saw all methods were 2 char so I also made this like that.

Reply:
yeah but prefer conistency over short cause we don't have java docs

Reply:
should i also rename "in" for "contains" (in python it is "contains")?

Reply:
no I actually think "contains" is bad, `x contains ["avx"]` that sounds incorrect, or? the condition is actually `x in ["a", "b"]